<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineupUpdate</class>
 <widget class="QWidget" name="LineupUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>阵容名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>大剑：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="swordEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="swordPlusButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="swordMinusButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>大棒：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rodEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rodPlusButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rodMinusButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bowMinusButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>攻速：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bowEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bowPlusButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>水滴：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tearEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tearPlusButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tearMinusButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shovelPlusButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>金铲铲：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shovelEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="shovelMinusButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vestMinusButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="beltEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="glovesPlusButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>魔抗：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cloakEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vestEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>拳套：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="glovesEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>护甲：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>腰带：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloakPlusButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vestPlusButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloakMinusButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="beltPlusButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="beltMinusButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="glovesMinusButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>781</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
